css/amp.css

```diff
@@ -621,6 +621,27 @@ i-amphtml-scroll-container.amp-active, i-amp-scroll-container.amp-active {
   visibility: visible;
 }
 
+amp-list[load-more] > [load-more-button] > .i-amphtml-loader {
+  visibility: hidden;
+}
+
+amp-list[load-more] > [load-more-loading],
+amp-list[load-more] > [load-more-failed],
+amp-list[load-more] > [load-more-button],
+amp-list[load-more] > [load-more-failed],
+amp-list[load-more] > [load-more-end] {
+  visibility: hidden;
+  position: absolute;
+  bottom: 0;
+}
+
+amp-list[load-more] > [load-more-loading].amp-visible,
```

Comment:
Is there a use-case where these elements needs to be visible before the `amp-list` script loads? If not, this rule can be moved to the component CSS.

Reply:
Great catch. This looks like the result of a bad rebase.